common/discordutil.js

```diff
@@ -545,6 +557,16 @@ module.exports = {
 
     const roleToRemove = newGuild.roles.cache.find(r => r.name.toLowerCase() === removedRole);
     guildMember.roles.remove(roleToRemove.id);
+  },
+
+  getNonBAMembers(interaction, client) {
+    const mainGuild = this.getAllGuilds([guildId], interaction, client)[0];
+    const satelliteGuilds = this.getAllGuilds(SATELLITE_GUILD_IDS_WITHOUT_BAE, interaction, client);
+    const nonBAMemberIds = [];
+    satelliteGuilds.forEach(satelliteGuild => {
+      satelliteGuild.members.cache.filter(m => (!mainGuild.members.cache.get(m.id) && !nonBAMemberIds.includes(m.id))).forEach(m => nonBAMemberIds.push(m.id));
```

Comment:
Maybe filter bots, in case some servers had bots that the other servers don't have? 🤔

```suggestion
      satelliteGuild.members.cache.filter(m => (!m.user.bot && !mainGuild.members.cache.get(m.id) && !nonBAMemberIds.includes(m.id))).forEach(m => nonBAMemberIds.push(m.id));
```